L/libigc/build_tarballs.jl

```diff
@@ -101,7 +90,6 @@ ninja -C build -j ${nproc} install
 platforms = [
     Linux(:i686, libc=:glibc),
     Linux(:x86_64, libc=:glibc),
-    Linux(:x86_64, libc=:musl),
```

Comment:
Was it failing?

Reply:
Yeah, `Host compiler must support std::atomic` as part of the nested LLVM build. Didn't feel like fixing it, since `ligigc` is currently only used by NEO which doesn't support musl anyway.

Reply:
Ok, I thought it was related to recent musl issues that should have been fixed by https://github.com/JuliaPackaging/BinaryBuilderBase.jl/pull/27.  I was just wondering whether it was still the same issue

Reply:
I can try; have those shards been released already, or how do I test them?

Reply:
It's already here on Yggdrasil since yesterday.  The old error was that the include paths of x86_64-linux-musl were wrong and the compiler couldn't find the system header files.  The error you showed looks unrelated.